cirq-google/cirq_google/serialization/op_deserializer.py

```diff
@@ -31,13 +31,12 @@ class OpDeserializer(abc.ABC):
 
     @property
     @abc.abstractmethod
-    def serialized_id(self) -> str:
```

Comment:
Please update class docstring which refers to `serializer_id`.

Reply:
Done.  Also warnings suppressed.